plugin.php

```diff
@@ -318,6 +318,12 @@ function rest_validate_request_arg( $value, $request, $param ) {
 			}
 		}
 
+		if ( isset( $args['minimum'] ) && isset( $args['maximum'] ) ) {
```

Comment:
It feels like we are just inventing this adhoc? What does it mean to have `[ "type" => "string", "minimum" => 1 ]`? For now, I'd be more in favour of just validating these in their own callback rather than abstracting `rest_validate_request_arg` to handle it until we have a solid spec on how to achieve this.